src/components/BaseAvatar.vue

```diff
@@ -32,14 +32,14 @@ watch(
     <img
       :src="
         imgUrl ||
-        `https://stamp.fyi/avatar/eth:${address}?s=${parseInt(size) * 2}`
+        `https://stamp.fyi/avatar/eth:${address}?s=${(parseInt(size as string) || 22) * 2}`
```

Comment:
Is `as string` really needed here? `size` prop is string anyway.
We can also [define defaults](https://vuejs.org/api/sfc-script-setup.html#default-props-values-when-using-type-declaration), instead of `|| 22`.

And btw... why `* 2`? 🤔 

```suggestion
        `https://stamp.fyi/avatar/eth:${address}?s=${(parseInt(size) || 22) * 2}`
```

Reply:
I agree, define defaults, and this will take care of handling the || 22